doc/api/esm.md

```diff
@@ -431,6 +431,46 @@ thrown:
 }
 ```
 
+#### Self-referencing a package using its name
+
+Self-referencing a package's name in an `import` is allowed. For example,
+let's assume the `package.json` is:
+
+```json
+// package.json
+{
+  "name": "a-package",
+  "exports": {
+    ".": "./main.mjs",
+    "/cjs": "./main.js"
+  }
+}
+```
+
+Then any module _in that package_ can reference an export in the package itself:
+
+```js
+// ./a-module.mjs
+import { something } from 'a-package'; // Imports "something" from ./main.mjs.
+```
+
+Self-referencing is available only if `package.json` has `exports`, and will
+allow importing only what that `export` allows. So the code below,
```

Comment:
Do you mean “what `"exports"` allows”, or what the `export` keyword provides as a named export?

Reply:
You're right. It's confusing. I meant what the `exports` allows. I will make it plural, and add some more wording to make it more explicit.

Reply:
Fixed.

Reply:
Always writing it as `"exports"` (with the quotes) can also make it clearer that it’s not a typo for `export`.